lua/pluginList.lua

```diff
@@ -324,4 +324,38 @@ return packer.startup(function()
          require("mappings").fugitive()
       end,
    }
+
+   use {
+     "glepnir/lspsaga.nvim",
+     after = "nvim-lspconfig",
+     config = function ()
+       require("plugins.others").saga()
+     end,
+     setup = function ()
+       require("mappings").saga()
+     end
+   }
+
+  use {
```

Comment:
It looks like `Hop` has some notable issues with the current NeoVim stable release,
advanced Vim motions are really cool and I like the idea.....but I don't think `Hop` is ready for use due to bugs.

[See first paragraphs of it's README](https://github.com/phaazon/hop.nvim#readme)

`it suffers from a few drawbacks making it not comfortable to use with Neovim version >0.5 – at least at the time of writing these lines:`